### PR TITLE
docs: adding section on where to find themes

### PR DESCRIPTION
**Pull request by Lucas-C**
\+ mention Pypi classifiers

Resulting theme page preview:

### CODE REVIEW

**Comment by Lucas-C** [comment]
I could also mention [pelican-plugin](https://github.com/topics/pelican-plugin) & [pelican-plugins](https://github.com/topics/pelican-plugins) GitHub topics.

**Comment by Lucas-C** [comment]
@getpelican/reviewers is anyone willing to review this please?

File: docs/themes.rst [hunk 1]
```diff
@@ -1,7 +1,26 @@
 .. _theming-pelican:
 
+Themes
+######
+
+Where to find them?
+*******************
+Namespace themes can be found in the `pelican-themes repository`_.
```
Comment (avaris) [inline docs/themes.rst:8]: "Namespace themes"? I don't think we have those (yet :))

**Comment by justinmayer** [comment]
Thanks for the submission, Lucas. I'm included to take a simpler approach and thus submitted an alternative set of changes in #2933.

**Comment by justinmayer** [comment]
@Lucas-C: Thanks again for pointing out how we could improve things here. Much appreciated!

**Pull request closed without merging**